assets/app/view/game/tile.rb

```diff
@@ -39,6 +39,9 @@ def should_render_revenue?
 
         return false if revenue.uniq.size > 1
 
+        # avoid obcsuring track with revenues
+        return true if @tile.cities.empty? && @tile.city_towns.size == 2 && @tile.exits.size > 4
```

Comment:
> 
> 
> > > When a tile has over two towns all with the same revenue, only render a single revenue
> > 
> > 
> > Your wording here sounds like the intent is for this to apply only to tiles with three or more towns, but in the screenshots, it applies to two or more, e.g., `780_2b`.
> 
> Yeah, bad comment.


This is the LOC that makes this make sense now, I just noticed that `16_2` shows two circles, so I was confused about `780_2b` only showing one. So it's more like "When a tile has towns with 3+ towns all with the same revenue, or 2 towns and a bunch of track, only render a single revenue"